### ISSUE TITLE
Sift Science Scores Change Upon Refresh

### ISSUE THREAD

**Issue by Fidyyuan** [Fermiac/woocommerce-siftscience#46]
Seems the Sift Science scores in Woocommerce are constantly changing upon refresh and don't match up to the Siftscience.com scores. The data seems to be sent properly to Siftscience.com though.

**Comment by nabsul** [comment]
I double checked this and it seems to be working fine. Please note the following:

- Make sure you're using the correct sandbox/production keys in the plugin and the dashboard in siftscience.com is correctly set to sandbox/production.
- I've observed that scores tend to change periodically. I assume Sift Science is running background machine learning tasks that might change the score from time to time.

I'm going to close this issue for now. If the advice above didn't help, please re-open it, and perhaps add some more details like screen shots of the wp-admin and sift science dashboards.

Thanks!

**Comment by Fidyyuan** [comment]
It doesn't seem to be matching up quite right. I've made sure that the Sift Science account is setup as Production. I've set the correct Rest API key and Javascript Snippet Key in the plugin (Sift Science says they're both active too). 

I've attached a screenshot of an order's score and the user's score in Sift Science. 
![capture](https://cloud.githubusercontent.com/assets/385291/18795006/59d7c67c-8190-11e6-9f3f-1433e8b3636e.PNG)
![user 1](https://cloud.githubusercontent.com/assets/385291/18795007/59d999d4-8190-11e6-926a-8133a2aad4f5.PNG)
